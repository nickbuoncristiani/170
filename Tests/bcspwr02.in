49
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49
5 37 49 26 13 42 40 47 19 30 33 36 29 2 7 1 4 48 16 24 6 11 17 3 14
43
x 1.86012 1.1216 1.9628 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.86012 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.1216 x x x 1.99856 1.61624 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.9628 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.99856 x x x x x x x x x x x x x x x x x x x x x x x x x 1.34777 x x x x x x x x x x x x x x x x x x x x
x x 1.61624 x x x x x 1.93312 x x x x x x x 1.58539 x x x x x x x x x x x 1.38094 x x x x 1.15739 x x x x x x x x x x x x x x x
x x x x x x x 1.10274 1.55586 x x x x x 1.77922 x x x x x x x x x x x x x x x x x x x x x x x 1.76661 x 1.90616 x x x x x x x x
x x x x x x 1.10274 x 1.68761 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x 1.93312 1.55586 1.68761 x x x 1.16896 x x x 1.33596 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.56529 x x x x x x 1.25576 x x x 1.83515 1.56451 x x 1.71359 x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.56529 x 1.92176 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.16896 x 1.92176 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.60815 x x x x x x x
x x x x x x x x x x x x x 1.74748 x x 1.56845 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x 1.74748 x 1.04409 1.36568 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.77922 x x x x x x 1.04409 x x x x x x x x x x x x x x x x 1.22396 x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.33596 x x x x 1.36568 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x 1.58539 x x x x x x 1.56845 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.25576 x x x x x x x x 1.89107 1.99199 1.9584 x x x 1.78745 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.89107 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.99199 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.9584 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.83515 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.56451 x x x x x x x x x x x x x 1.05581 x x 1.32407 x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.05581 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.78745 x x x x x x x x x x x x x x x x x x x x x x x 1.33365 x x 1.82086 x x x x
x x x x x x x x x 1.71359 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.31394 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.32407 x x x x x x x x 1.2524 1.13781 1.85155 x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.42378 1.80411 x x x x x x x x x x x x x x x x x x x
x x x x 1.34777 1.38094 x x x x x x x x x x x x x x x x x x x x x 1.42378 x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.80411 x x x 1.82916 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.22396 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.2524 x x 1.82916 x x x x x x x 1.33392 x x x x x x x x 1.63834 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.13781 x x x x x x 1.93337 1.93981 1.83493 1.30789 1.23095 x x x x x x x x x x x
x x x x x 1.15739 x x x x x x x x x x x x x x x x x x x x 1.85155 x x x x x 1.93337 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.93981 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83493 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.30789 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.33392 1.23095 x x x x x x x x x x 1.16722 x x 1.82061 x x
x x x x x x 1.76661 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.73541 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.73541 x x x x x x x x x x
x x x x x x 1.90616 x x x x x x x x x x x x x x x x x x 1.31394 x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.60815 x x x x x x x x x x x x 1.33365 x x x x x x x x x x x x x x x x x x x x 1.34481 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.66085 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.16722 x x x x 1.66085 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x 1.82086 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.34481 x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.63834 x x x x x 1.82061 x x x x x x x x x 1.25991 1.37052
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.25991 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.37052 x x
