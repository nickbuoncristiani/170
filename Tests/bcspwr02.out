43 44 38 33 34 6 3 1 3 5 29 6 17 13 14 16 9 7 41 26 10 11 12 42 25 18 10 23 27 32 47 32 38 44 43
17
33 33 36 37
6 6
3 3
1 1 2 4
5 5
29 29 30
17 17
13 13
14 14
16 16
7 7 40
26 26
11 11
42 42
18 19
32 24
47 49 47 48
